### PR TITLE
Add a brief note about Group Blocking in Dex ext

### PR DESCRIPTION
**Pull request by curche**

### CODE REVIEW

File: src/help/faq/README.md [hunk 1]
```diff
@@ -252,6 +252,39 @@ You need to [migrate](./source-migration.md) all your MangaDex manga from MangaD
 ##### Why can I not restore from a JSON backup?
 JSON backups are now unusable due to the ID change. You will have to manually re-add your manga.
 
+#### How can I block particular Scanlator Groups?
+
+Mangadex extensions now allows you to block Scanlator Groups. Chapters uploaded by a Blocked Scanlator Group will not show up in Latest or in Manga chapters list. For now, you can only block Groups by entering their UUIDs manually.
```
Comment (Soitora) [inline src/help/faq/README.md:257]:
```suggestion
The **MangaDex** extension now allows you to block **Scanlator Groups**. Chapters uploaded by a **Blocked** Scanlator Group will not show up in Latest or in Manga chapters list. For now, you can only block Groups by entering their UUIDs manually. 
```
Maybe something like this? Not sure but at least capitalize MangaDex

File: src/help/faq/README.md [hunk 2]
```diff
@@ -252,6 +252,39 @@ You need to [migrate](./source-migration.md) all your MangaDex manga from MangaD
...
+       * The Group *Tristan's test scans* has the URL 
+	       - [https://mangadex.org/group/6410209a-0f39-4f51-a139-bc559ad61a4f/tristan-s-test-scans](https://mangadex.org/group/6410209a-0f39-4f51-a139-bc559ad61a4f/tristan-s-test-scans)
+           - Therefore, their UUID will be `6410209a-0f39-4f51-a139-bc559ad61a4f`
+       * Other Examples include:
+	       + Azuki Manga     | `5fed0576-8b94-4f9a-b6a7-08eecd69800d`
+           + Bilibili Comics | `06a9fecb-b608-4f19-b93c-7caab06b7f44`
+           + Comikey         | `8d8ecf83-8d42-4f8c-add8-60963f9f28d9`
+           + MangaPlus       | `4f1de6a2-f0c5-4ac5-bce5-02c7dbb67deb`
+
+B. Enter the Details to the Tachiyomi Mangadex extension `v1.2.150+`:
```
Comment (Soitora) [inline src/help/faq/README.md:274]: Capitalize the name, also, maybe change to something like "Blocking a group using their UUID ..."

File: src/help/faq/README.md (hunk 1 above)
Comment (curche) [reply src/help/faq/README.md]: Makes sense. Done

**Review by Soitora — APPROVED** [review]
Good enough I guess, I'm not good enough at content to be more nitpicky than this

**Pull request merged**